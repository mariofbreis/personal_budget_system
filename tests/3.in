i 60 0 10 r Receita0
i 60 1 10 r Receita1
i 60 2 10 r Receita2
i 60 3 10 r Receita3
i 60 4 10 r Receita4
i 60 5 10 r Receita5
i 60 6 10 r Receita6
i 60 7 10 r Receita7
i 60 8 10 r Receita8
i 60 9 10 r Receita9
i 60 10 10 r Receita10
i 60 11 10 r Receita11
i 60 12 10 r Receita12
i 60 13 10 r Receita13
i 60 14 10 r Receita14
i 60 15 10 r Receita15
i 60 16 10 r Receita16
i 60 17 10 r Receita17
i 60 18 10 r Receita18
i 60 19 10 r Receita19
i 60 20 10 r Receita20
l 0
l 1
l 2
l 3
l 4
l 5
l 6
l 7
l 8
l 9
l 10
l 11
l 12
l 13
l 14
l 15
l 16
l 17
l 18
l 19
l 20
l 21
l 22
l 23
l 24
l 25
l 26
l 27
l 28
l 29
l 30
l 31
l 32
l 33
l 34
l 35
l 36
l 37
l 38
l 39
l 40
l 41
l 42
l 43
l 44
l 45
l 46
l 47
l 48
l 49
l 50
l 51
l 52
l 53
l 54
l 55
l 56
l 57
l 58
l 59
l 60
l 61
l 62
l 63
l 64
l 65
l 66
l 67
l 68
l 69
l 70
l 71
l 72
l 73
l 74
l 75
l 76
l 77
l 78
l 79
l 80
l 81
l 82
l 83
l 84
l 85
l 86
l 87
l 88
l 89
l 90
l 91
l 92
l 93
l 94
l 95
l 96
l 97
l 98
l 99
l 100
