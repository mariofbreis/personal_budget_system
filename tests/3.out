0 60 r Receita0
0 60 r Receita1
0 60 r Receita2
0 60 r Receita3
0 60 r Receita4
0 60 r Receita5
0 60 r Receita6
0 60 r Receita7
0 60 r Receita8
0 60 r Receita9
0 60 r Receita0
1 60 r Receita10
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
0 60 r Receita1
1 60 r Receita11
0 60 r Receita2
1 60 r Receita12
0 60 r Receita3
1 60 r Receita13
0 60 r Receita4
1 60 r Receita14
0 60 r Receita5
1 60 r Receita15
0 60 r Receita6
1 60 r Receita16
0 60 r Receita7
1 60 r Receita17
0 60 r Receita8
1 60 r Receita18
0 60 r Receita9
1 60 r Receita19
0 60 r Receita0
1 60 r Receita10
2 60 r Receita20
